src/flat-button.jsx

```diff
@@ -197,7 +258,7 @@ const FlatButton = React.createClass({
         onKeyboardFocus={this._handleKeyboardFocus}
         onMouseLeave={this._handleMouseLeave}
         onMouseEnter={this._handleMouseEnter}
-        onTouchStart={this._handleTouchStart}
```

Comment:
I console.log to check if it was ever called, but it never was. Also, enhanced button had prop onTouchTap and not onTouchStart, so I changed it. Not sure if onTouchStart is specific for mobile?